### PR TITLE
Add a tutorial to train the model in DeepCTR using PS

### PR DESCRIPTION
**Pull request by workingloong**

### CODE REVIEW

File: docs/tutorials/elasticdl_deepctr.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
+# ElasticDL：Distributed Train the DeepCTR Model on Kubernetes
+
+This document shows how to use ElasticDL to train the premade Keras Model
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr.md]: keras Models

File: docs/tutorials/elasticdl_deepctr.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+                shard = index // records_per_shard
+                file_path_name = os.path.join(dir_name, "data-%05d" % shard)
+                writer = recordio.Writer(file_path_name)
+            example = convert_data_to_tf_example(row, COLUMNS)
+            writer.write(example)
+
+        if writer:
+            writer.close()
+
+# We use the same dataset for training and valuation for simplity.
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr.md]: valuation -> evaluation

File: docs/tutorials/elasticdl_deepctr.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+convert_to_recordio_files("./criteo_sample.txt", "./data/criteo_records/train")
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr.md:88]: We convert the same file into train&test dataset?

File: docs/tutorials/elasticdl_deepctr.md [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+import os
+import pathlib
+import sys
+
+import recordio
+import tensorflow as tf
+
+COLUMNS = (
+    ["label"]
+    + ["I" + str(i) for i in range(1, 14)]
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr.md]: ["I" + str(i) for i in range(1, 14)] => [("I" + str(i)) for i in range(1, 14)]
for readability.

File: docs/tutorials/elasticdl_deepctr.md [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+```
+
+If we want to submit a training job on ACK of Aliyun or GKE
+of Google Cloud, we need to upload RecordIO files to the
+PersistentVolume of the cluster. Then, we can submit a training
+job like the above command on Minikube.
+
+### Check Job Status
+
+After the job submission, we can run the command kubectl get pods to list
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr.md]: kubectl get pods => `kubectl get pods`

File: docs/tutorials/elasticdl_deepctr.md (hunk 3 above)
Comment (workingloong) [reply docs/tutorials/elasticdl_deepctr.md:88]: Yes. We only want to show that users can set training and validation datasets using different folders.

File: docs/tutorials/elasticdl_deepctr.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+
+        if writer:
+            writer.close()
+
+# We use the same dataset for training and evaluation for simplity.
+convert_to_recordio_files("./criteo_sample.txt", "./data/criteo_records/train")
+convert_to_recordio_files("./criteo_sample.txt", "./data/criteo_records/test")
+```
+
+Except RecordIO format, ElasticDL also supports other storage with the index
```
Comment (brightcoder01) [inline docs/tutorials/elasticdl_deepctr.md]: Except -> Besides
storage -> storages

File: docs/tutorials/elasticdl_deepctr.md (hunk 4 above)
Comment (brightcoder01) [reply docs/tutorials/elasticdl_deepctr.md]: It seems that these pieces of codes are not updated.

**Review by brightcoder01 — APPROVED** [review]

**Pull request merged**